### ISSUE TITLE
/(h)pyramid is disabled

### ISSUE THREAD

**Issue by aggelosQQ** [ImmaFreedom-Dev/IF-WorldEdit#1]
Can we enable //pyramid and //hpyramid? I cannot find how to.

@tylerhyperHD @OxLemonxO @OutOfYourMind @LegendIsAwesomes

**Comment by aggelosQQ** [comment]
//sphere aswell

**Comment by tylerhyperHD** [comment]
Give me a valid reason to unblock it and I will consider it.

**Comment by tylerhyperHD** [comment]
Re-assigned to new issue, closing this one as it has been fixed already.
